bokehjs/src/less/legend.less

```diff
@@ -1,7 +1,6 @@
 :host {
   position: relative;
   width: max-content;
```

Comment:
Should I also remove `width: max-content`?

Reply:
(Or to ask it another way, is there any reason I shouldn't?)

Reply:
Probably we could. There's some excessive CSS there to make things work in Firefox, which proved to be quite problematic when complex nested flex and grids layouts are intermixed. Given the test coverage, I think you can remove it, run tests and if nothing fails, then we can safely assume it's fine.